h/static/scripts/group-forms/components/test/CreateEditGroupForm-test.js

```diff
@@ -249,10 +137,10 @@ describe('CreateEditGroupForm', () => {
 
   it('does not warn when leaving page if there are unsaved changes', () => {
     const { elements } = createWrapper();
-    const nameEl = elements.name.fieldEl;
 
-    nameEl.getDOMNode().value = 'modified';
-    nameEl.simulate('input');
+    act(() => {
+      elements.nameField.prop('onChangeValue')('modified');
+    });
```

Comment:
`act` invokes its callback and then synchronously flushes state updates and effects. Previously `simulate` did this implicitly.